db.go

```diff
@@ -252,6 +254,9 @@ func Open(opt Options) (*DB, error) {
 		bannedNamespaces: &lockedKeys{keys: make(map[uint64]struct{})},
 		threshold:        initVlogThreshold(&opt),
 	}
+
+	db.setSyncChan(opt.syncChan)
```

Comment:
I am thinking if it seems fine to directly do db.syncChan = chan in the test code and skip this code here

Reply:
Sadly, this does not work. We actually want to capture some of the messages that occur during the opening of the DB (i.e., inside of Open(opts)). In particular, the vlog.init message occurs during the Open function, and if we wait until after Open returns the DB, we will have missed the opportunity to capture the message.